emulator/scripts/build.sh

```diff
@@ -8,6 +8,7 @@ cp /app/config.ini /app/arcturus/target/config.ini
 mkdir /app/arcturus/target/plugins
 cd /app/arcturus/target/plugins
 wget https://cdn.discordapp.com/attachments/641088820598538252/873759980329766942/NitroWebsockets-3.1.jar
+#wget https://git.krews.org/Raizer/cosmic-assets/-/raw/master/Plugin/Webkit.jar
```

Comment:
You need to remove this, it's useless ! :3

Reply:
Yup

Reply:
Tbh, I had several problems with the database (tables already existing making the import crash) so I'll take a look at it again to make it clean.

Reply:
Solved in https://github.com/Holo5/nitro-docker/pull/3/commits/59e3706b0e3c75bf3ed12c4e7e098e2ba0bda99f